Run3Detector/analysis/utilities/milliqanProcessor.py

```diff
@@ -10,14 +12,96 @@
 
 class milliqanProcessor():
 
-    def __init__(self, filelist, branches, schedule=None, cuts=None, plotter=None, max_events=None):
+    def __init__(self, filelist, branches, schedule=None, cuts=None, plotter=None, max_events=None, runQualityOverride=False, qualityLevel="tight"):
+        self.qualityLevelString = qualityLevel
```

Comment:
can runQualityOverride and qualityLevel be combined? It would be better to have just one setting